### DISCUSSION TITLE
getting a issue i am getting these two extra lines above my output in terminal can u plz resolve it

### DISCUSSION THREAD

**Discussion by shard12** [microsoft/vscode-discussions#1190] [Extension Development QnA]
![● hello java - Visual Studio Code 5_13_2024 8_58_31 PM](https://github.com/microsoft/vscode-discussions/assets/168275463/66b07d6a-f4c3-43e1-8328-8cdc86dfd8ba)
![hello java - Visual Studio Code 5_13_2024 1_30_14 PM](https://github.com/microsoft/vscode-discussions/assets/168275463/6b86c72f-e0b1-4a82-b375-5f391c49720c)

**Comment by gjsjohnmurray** [comment]
Unless you are developing an extension for VS Code this is not the right place for your question.

**Comment by abrilurena** [comment]
Hi, @shard12! do you still need help with this?

**Reply by shard12** [reply]
yes
